clients/src/test/java/org/apache/kafka/common/internals/SecurityManagerCompatibilityTest.java

```diff
@@ -34,7 +34,7 @@
 
 public class SecurityManagerCompatibilityTest {
 
-    @EnabledForJreRange(min = JRE.JAVA_8, max = JRE.JAVA_22)
+    @EnabledForJreRange(max = JRE.JAVA_22)
```

Comment:
there are two usages having min=18 in this file. Could you please cleanup them also?

Reply:
Would removing min=18 cause the JDK 11 and JDK 17 versions to run, which we do not expect?

Reply:
you are right :)

Reply:
Thanks for reviewing ^^